Makefile

```diff
@@ -43,9 +46,8 @@ endif
 declare-testnet:
```

Comment:
Leave a comment saying in the future we would need to add the NFTManager here as well